ui/src/pages/index.tsx

```diff
@@ -15,38 +12,21 @@ export const getServerSideProps = async () => {
 export default function Index({
   markets,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
-  const router = useRouter()
-  const [marketTitle, setMarketTitle] = useState('')
-  async function handleSubmit(e: FormEvent<HTMLFormElement>) {
-    e.preventDefault()
-    const res = await api.postMarkets({ title: marketTitle })
-    console.log(res)
-    router.reload() // タスク一覧の更新が面倒なのでリロードしてごまかしている
-  }
   return (
     <>
-      <div className="p-4 font-bold">Hello Tailwind CSS and daisyUI!</div>
-      <button className="btn btn-primary">Button</button>
-      <SampleCounter />
-      <hr />
-      <div className="p-5 border rounded">
-        <p>Markets</p>
-        <ul>
-          {markets.map((market) => {
-            return <li key={market.id}>・{market.title}</li>
-          })}
-        </ul>
-        <form onSubmit={handleSubmit} className="mt-5">
-          <input
-            type="text"
-            placeholder="type your market title ..."
-            className="input input-bordered input-sm mr-2"
-            value={marketTitle}
-            onChange={(e) => setMarketTitle(e.target.value)}
-          />
-          <button className="btn btn-sm">Add</button>
-        </form>
-      </div>
+      <AppHero />
+      <p className="font-bold mt-8">Popular Markets</p>
+      <ul className="grid grid-cols-3 gap-10 mt-4">
+        {markets.map((market) => {
+          return (
+            <Link key={market.id} href={`markets/${market.contract}`}>
```

Comment:
細かいんですが、以下のようにしないとコンソールでエラーが出てしまうようです！
```suggestion
            <Link key={market.id} href={`/markets/${market.contract}`}>
```